migrations/versions/b389abb05262_v1_0_rc_1.py

```diff
@@ -47,12 +47,13 @@ def upgrade():
     sa.Column('created_at', sa.DateTime(), nullable=True),
     sa.Column('id', sa.Integer(), nullable=False),
     sa.Column('email', sa.String(length=320), nullable=False),
-    sa.Column('full_name', sa.String(length=120), nullable=False),
+    sa.Column('experiment', sa.String(length=120), nullable=True),
     sa.Column('institution', sa.String(length=120), nullable=True),
-    sa.Column('key', sa.String(length=120), nullable=False),
+    sa.Column('name', sa.String(length=120), nullable=False),
     sa.Column('pending', sa.Boolean(), nullable=True),
-    sa.Column('experiment', sa.String(length=120), nullable=True),
+    sa.Column('refresh_token', sa.Text, nullable=False, unique=True),
     sa.Column('updated_at', sa.DateTime(), nullable=True),
+    sa.Column('sub', sa.String(120), nullable=False, unique=True, index=True),
```

Comment:
This is messed up, there should be a v1.0.0.0-RC3 database version.. don't try to change released db schema.

Reply:
Didn't really know what it was but I'll take a look at the Alembic docs

Reply:
Okay, this should be fixed now.. sorry about that.